src/seed/seed_db.ts

```diff
@@ -734,22 +739,56 @@ async function reloadAutocompleteData(): Promise<void> {
             const currentDate = new Date();
             if (songsRemoved.length) {
                 logger.info(`${songsRemoved.length} songs removed.`);
-                await sendDebugAlertFileWebhook(
-                    discordDateFormat(currentDate, "f"),
+                const description = `**${songsRemoved.length} songs removed**:\n${songsRemoved.join("\n")}`;
+
+                await sendInfoWebhook(
                     process.env.SONG_UPDATES_WEBHOOK_URL!,
-                    `${songsRemoved.length} songs removed:\n${songsRemoved.join("\n")}`,
-                    "removed_songs.txt",
+                    discordDateFormat(currentDate, "f"),
+                    truncatedString(
+                        description,
+                        EMBED_DESCRIPTION_MAX_LENGTH - 1,
+                    ),
+                    EMBED_ERROR_COLOR,
+                    KmqImages.NOT_IMPRESSED,
+                    undefined,
                 );
+
+                if (description.length >= EMBED_DESCRIPTION_MAX_LENGTH) {
```

Comment:
Should we fall back to only the file upload when truncated instead of duplicating data?

Reply:
Is this not what its doing?

Reply:
I believe this is sending an embed with the truncated songs, and then sending the full list as a file. We could send only the file if it's too long, similar to how ,list works

Reply:
Would rather still show some/most of the list in the original message, since mobile users cant see file previews at all

Reply:
Good point, let's leave the current implementation